stable/sonarqube/values.yaml

```diff
@@ -207,3 +207,11 @@ mysql:
 # podLabels:
 #   key: value
 podLabels: {}
+
+init:
+  sysctl:
+    # Set to false to disable the privileged running sysctl init container .
+    # When disabling the init container the nodeSelector must only select Elasticseach compatible nodes.
+    # When enabled sets vm.max_map_count=262144 which is the minimum Elasticsearch requires.
+    enabled: true 
```

Comment:
Could you fix below on line 216?

stable/sonarqube/values.yaml
  216:18    error    trailing spaces  (trailing-spaces)

Reply:
I removed the trailing whitespace. For some reason `helm lint` does not report any issues.
Maybe I should just use `ct lint` for linting locally.